app/Controllers/FrontController.php

```diff
@@ -59,8 +61,9 @@ public function main($table = null, $action = null, $id = null)
         $this->table = $table;
         $this->action = $action;
         $this->id = $id;
-
-        $this->businessModel = new BusinessModel($this->table, $this->action, $this->id);
+        
+        $businessModel = $this->businessModel ;
+        $this->businessModel = new $businessModel($this->table, $this->action, $this->id);
```

Comment:
Ok, got you. It goes hand in hand with the dynamic `FrontController`. Merging now...